scripts/json2db.pl

```diff
@@ -82,9 +99,13 @@
 
 sub process_motions {
     my ($policy, $dreamid) = @_;
+    $start_transaction->execute();
```

Comment:
Bit confused why this file calls transaction SQL, whereas another one you've changed you've altered the AutoCommit value. Best to do both the same way, unless there's a reason?

Reply:
I've switched to this to the AutoCommit approach used elsewhere.  Run the import and it seems fine.